fsharp-backend/src/LibExecution/StdLib/LibFloat.fs

```diff
@@ -1,317 +1,317 @@
-          | args ->
-              incorrectArgs ())
-    ; sqlSpec = NotYetImplementedTODO
-      ; previewable = Pure
-    ; deprecated = NotDeprecated }
-  ; { name = fn "Float" "min" 0
-
-    ; parameters = [Param.make "a" TFloat; Param.make "b" TFloat]
-    ; returnType = TFloat
-    ; description = "Returns the lesser of float `a` and float `b`"
-    ; fn =
-
-          (function
-          | _, [DFloat a; DFloat b] ->
-              DFloat (Float.min a b)
-          | args ->
-              incorrectArgs ())
-    ; sqlSpec = NotYetImplementedTODO
-      ; previewable = Pure
-    ; deprecated = NotDeprecated }
-  ; { name = fn "Float" "max" 0
-
-    ; parameters = [Param.make "a" TFloat; Param.make "b" TFloat]
-    ; returnType = TFloat
-    ; description = "Returns the greater of float `a` and float `b`"
-    ; fn =
-
-          (function
-          | _, [DFloat a; DFloat b] ->
-              DFloat (Float.max a b)
-          | args ->
-              incorrectArgs ())
-    ; sqlSpec = NotYetImplementedTODO
-      ; previewable = Pure
-    ; deprecated = NotDeprecated }
-  ; { name = fn "Float" "clamp" 0
-
-    ; parameters = [Param.make "value" TFloat; Param.make "limitA" TFloat; Param.make "limitB" TFloat]
-    ; returnType = TFloat
-    ; description =
-        "If `value` is within the range given by `limitA` and `limitB`, returns `value`.
-         If `value` is outside the range, returns `limitA` or `limitB`, whichever is closer to `value`.
-         `limitA` and `limitB` can be provided in any order."
-    ; fn =
-
-          (function
-          | _, [DFloat v; DFloat a; DFloat b] ->
-              let min, max = if a < b then (a, b) else (b, a) in
-              ( match Float.clamp v ~min ~max with
-              | Ok clamped ->
-                  DFloat clamped
-              | Error e ->
-                  (* Since min and max are pre-sorted, this can only happen if min or max are NaN.
-                   * TODO: eliminate NaNs so that this can't happen
-                   * (at time of writing (f86edaa1c58c94e27186060ae4fe8745112dd0e5), NaNs can't be parsed,
-                   * so this can't happen in practice) *)
-                  let info =
-                    [("a", Float.to_string a); ("b", Float.to_string b)]
-                  in
-                  Exception.code
-                    ~info
-                    ("Internal Float.clamp exception: " ^ Error.to_string_hum e)
-              )
-          | args ->
-              incorrectArgs ())
-    ; sqlSpec = NotYetImplementedTODO
-      ; previewable = Pure
-    ; deprecated = NotDeprecated } ]
+// let list_coerce ~(f : dval -> 'a option) (l : dval list) :
+//     ('a list, dval list * dval) Result.t =
+//   l
+//   |> List.map (fun dv ->
+//          match f dv with Some v -> Result.Ok v | None -> Result.Error (l, dv))
+//   |> Result.all
+
+
+// let ( >>| ) = Result.( >>| )
+
+let fns : List<BuiltInFn> =
+  [ { name = fn "Float" "ceiling" 0
```

Comment:
The old function had 2 names: "Float::ceiling" and "Float::roundUp" (its been mangled a bit by some regexes). So you need to add a function that does the same thing called Float::roundup_v0.